packages/beacon-node/src/chain/chain.ts

```diff
@@ -373,7 +373,7 @@ export class BeaconChain implements IBeaconChain {
     }
 
     if (metrics) {
-      metrics.opPool.attestationPoolSize.addCollect(() => this.onScrapeMetrics(metrics));
+      metrics.clockSlot.addCollect(() => this.onScrapeMetrics(metrics));
```

Comment:
don't see a reason why this needs to be attached to a op pool metric, so I rather used clock so we don't need to change this line in the future due to metric restructuring